packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Compact.Example.tsx

```diff
@@ -1,13 +1,23 @@
+// @codepen
+
 import * as React from 'react';
 import { TextField } from 'office-ui-fabric-react/lib/TextField';
 import { DetailsList, DetailsListLayoutMode, Selection } from 'office-ui-fabric-react/lib/DetailsList';
 import { MarqueeSelection } from 'office-ui-fabric-react/lib/MarqueeSelection';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
```

Comment:
I might be missing something obvious but I don't see the import being used anywhere?

Reply:
On the next line :)